src/circleci/rollcage/core.clj

```diff
@@ -151,20 +151,20 @@
 (defn- send-item-http
   "Send a Rollbar item using the HTTP REST API.
   Return the result JSON parsed as a Map"
-  [^String endpoint ^Throwable exception item]
+  [block-fields ^String endpoint ^Throwable exception item]
```

Comment:
The blocking of fields should happen higher up the callstack. I would put it into the `notify` function. 


We need to separate concerns so that we can test in isolation. 

This function is the function that sends items over HTTP, and should not be concerned with the contents of an item. `make-rollbar` returns an item in the shape expected by the Rollbar API.